docs/STYLEGUIDE.md

```diff
@@ -0,0 +1,65 @@
+# Style guide
+
+This style guide applies to all documentation created for Grafana products.
+
+## Contributing
+
+This style guide is a living document. Add to it whenever a style decision is made or question is answered.
+
+## Published guides
+
+For all items not covered in this guide, refer to the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/) and the [Chicago Manual of Style](https://www.chicagomanualofstyle.org/home.html).
+
+## Grafana-specific style
+
+### General
+
+* Use active voice. Avoid passive voice.
+  - Passive: The zombie was killed.
+  - Active: I killed the zombie.
+* Write in the imperative second person. Examples: You can do a thing. Click the panel. Shoot the zombie.
```

Comment:
I'd like to use a friendlier example than _shoot the zombie_, e.g. eat the cake, pet the dog.